plugin/trino-resource-group-managers/src/test/java/io/trino/plugin/resourcegroups/db/TestDbResourceGroupsFlywayMigration.java

```diff
@@ -24,26 +26,24 @@
 import static org.testng.Assert.assertEquals;
 
 @Test(singleThreaded = true)
-public class TestDbResourceGroupsFlywayMigration
+public abstract class TestDbResourceGroupsFlywayMigration
 {
-    private TestingMysqlServer mysqlServer;
-    private Jdbi jdbi;
+    protected JdbcDatabaseContainer<?> container;
+    protected Jdbi jdbi;
 
     @BeforeClass
     public final void setup()
     {
-        mysqlServer = new TestingMysqlServer()
-                .withDatabaseName("resource_groups")
-                .withUsername("test")
-                .withPassword("test");
-        mysqlServer.start();
-        jdbi = Jdbi.create(mysqlServer.getJdbcUrl(), mysqlServer.getUsername(), mysqlServer.getPassword());
+        startContainer();
```

Comment:
Nit: Instead of a contract that requires this method to have a side effect (initializing `container`), it could have a return type of `JdbcDatabaseContainer<?>` and the assignment would be visible here, in the super class.

Reply:
thanks @aalbu! I have applied this suggestion.